modules/tinymce/src/plugins/template/test/ts/module/InsertTemplate.ts

```diff
@@ -8,16 +8,37 @@ import Editor from 'tinymce/core/api/Editor';
 const toolbarButtonSelector = '[role="toolbar"] button[aria-label="Insert template"]';
 const dialogSelector = 'div.tox-dialog';
 
-const pInsertTemplate = async (editor: Editor, assertFn?: (elm: SugarElement<Node>) => Promise<void>): Promise<void> => {
+const waitUntilIframeLoaded = async (dialogEl: SugarElement<Node>): Promise<void> => {
+  await UiFinder.pWaitForState<HTMLIFrameElement>('iframe is loaded', dialogEl, 'iframe', (elm) => {
+    const iframeDoc = elm.dom.contentDocument || elm.dom.contentWindow?.document;
```

Comment:
Why are there two cases here will not `contentDocument` work on all browsers? If not add a comment why and on what.

Reply:
The code was there before my change. I assume this is because contentDocument isn't supported on pre- IE8 versions of IE. Added comment to document that.